### ISSUE TITLE
[BUG] DateTimeField. TAB does not move to next field if the TimeField part is hidden

### ISSUE THREAD

**Issue by ExtAnimal** [bryntum/support#12804]
### Description
Focus the startDate of a task in the task editor in SchedulerPro. Press TAB

If the issue cannot be reproduced in our demos, please provide a **minimal reproducible test case** (e.g. CodeSandbox, GitHub repo, or zipped project). Bug reports with clear reproduction steps or a test case are more likely to be fixed earlier than those without.

### Expected Behavior
Focus should move to the DateTimeField's next sibling if the TimeField part of the DateTimeField is hidden.

### Actual Behavior
The code attempts to focus the TimeField even if it hidden and calls preventDefault.
